ui/app/AppLayouts/Chat/views/private/MembersSelectorBase.qml

```diff
@@ -21,7 +21,7 @@ InlineSelectorPanel {
     property var rootStore
 
     readonly property int membersLimit: 20 // see: https://github.com/status-im/status-mobile/issues/13066
-    property bool limitReached: model.count >= membersLimit
+    property bool limitReached: (model.count === (membersLimit+1))
```

Comment:
Why "+1"? Members limit should include ourselves. Also `>=` is safer than `===`.